pulp-glue/pulp_glue/common/context.py

```diff
@@ -883,6 +883,18 @@ def modify(
             body["base_version"] = base_version
         return self.call("modify", parameters={self.HREF: href or self.pulp_href}, body=body)
 
+    def reclaim_space(
```

Comment:
Still unsure about this. But the command is not added to all the plugin sections.
It's just available on all the contexts where it is doomed to fail.

Reply:
We talked about this with @lubosmj futher and I think the current state is the best compromise.

You have to enter ```pulp repository reclaim-space ...``` , currently there is no way to call it in a way like ```pulp file repository reclaim-space ...``` but I believe it could be added easily if needed, since the method is in a class from which the plugin specific repository-sub-classes inherit (which wouldnt be possible if we created something like PulpCoreRepositoryContext to stop the inheritance). The individual plugins are also fairly differentiated between in the current state, since the user has to specify "file:file:repo" instead of "repo", which helps in avoiding mistakes.

Reply:
No you can't reclaim is only provided on the generic api endpoint. Any attempt to call this method on the PulpFileRepositoryContext will result in a key error looking up the corresponding operation in the api-spec.

Also this question must be answered with the glue-librarian-hat on. There is no arguing from the cli perspective.

Reply:
Can we just call it reclaim? Would there be any ambiguity?